src/main/java/com/squareup/kotlinpoet/TypeAliasSpec.kt

```diff
@@ -47,14 +50,16 @@ class TypeAliasSpec private constructor(builder: TypeAliasSpec.Builder) {
 
   fun toBuilder(): Builder {
     val builder = Builder(name, type)
-    builder.modifiers.addAll(modifiers)
+    builder.modifiers += modifiers
+    builder.typeVariables += typeVariables
     return builder
   }
 
   class Builder internal constructor(
       internal val name: String,
       internal val type: TypeName) {
-    internal var modifiers: MutableSet<KModifier> = mutableSetOf()
+    internal val modifiers = mutableSetOf<KModifier>()
```

Comment:
indent is off here

Reply:
It's not a param

Reply:
whops

Reply:
The problem is that `) {` should be on its own line

Reply:
...for the precise reason you just discovered

Reply:
definitely agree, want me to fix those?

Reply:
I didn't fix here to minimize the diff, but once this is approved and merged a PR to fix it would be great!

Reply:
Or fix it and i'll merge and rebase this since no one seems to be around...